src/sentry/static/sentry/app/utils/addForm.jsx

```diff
@@ -17,25 +17,28 @@ const createSearchIndex = field => {
 //   * what field(s) it matches:
 //     * what form group it belongs to
 //     * what route that belongs to
-const createSearchMap = ({route, formGroups, ...other}) => {
+const createSearchMap = ({route, formGroups, fields, ...other}) => {
+  let listOfFields = formGroups
+    ? flatMap(formGroups, formGroup => formGroup.fields)
+    : Object.keys(fields).map(fieldName => fields[fieldName]);
```

Comment:
isn't this the same as `Object.values(fields)`?

Reply:
Yeah, not sure if this is polyfilled though, `keys` has wider support

Reply:
We use it elsewhere, I think it's covered by `babel-polyfill`